apache/app.mako-dot-conf

```diff
@@ -38,6 +38,7 @@ Alias ${apache_base_path}/src ${apache_base_directory}/src
 Alias ${apache_base_path}/ ${apache_base_directory}/prd/
 RewriteRule ${apache_base_path}/src/layersConfig http:${api_url}/rest/services/all/MapServer/layersConfig [P]
 RewriteRule ${apache_base_path}/src/services http:${api_url}/rest/services [P]
+RewriteRule ${apache_base_path}/(?:src\/)?img\/maki/(.*) https:${api_url}/color/255,0,0/$1
```

Comment:
It's not needed for appcache , appcache doesn't save maki icons.

I will change the urls when parsing.